.ci/Jenkinsfile

```diff
@@ -62,7 +63,7 @@ pipeline {
       options { skipDefaultCheckout() }
       steps {
         cleanup()
-        withGoEnv(){
+        withGoEnv(version: "${GO_VERSION}"){
           dir("${BASE_DIR}"){
```

Comment:
```suggestion
        dir("${BASE_DIR}"){
          withGoEnv(version: "${GO_VERSION}"){
```

Reply:
Fixed in https://github.com/elastic/fleet-server/pull/346/commits/21328bf142b9578aa8ca73b4c912c0259cf03aa1